<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Insert</class>
 <widget class="QDockWidget" name="Insert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>300</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(64, 191, 100);
color:rgb(48, 48, 48);</string>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(51, 51, 51);
color:rgb(255, 255, 255);
</string>
    </property>
    <property name="title">
     <string>Insert Data</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome      </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nome">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(48, 48, 48);

</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Dominio  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_dominio">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(48, 48, 48);

</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Regno      </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_regno">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(48, 48, 48);

</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Divisione </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_divisione">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(48, 48, 48);

</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Costo        </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_costo">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(48, 48, 48);

</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Quantit√†  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_quantita">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(48, 48, 48);

</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(51, 51, 51);
color:rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
